apps/frontend/lib/disputes/wage-garnishment-shared.js

```diff
@@ -0,0 +1,235 @@
+/*eslint sort-keys: 0*/
+
+import debtTypes from "./dispute-types";
+import personalInfo from "./personal-info";
+import usaStates from "./usa-states";
+import yesnoSchema from "./yesno-schema";
+
+const schemas = {
+  json: {
+    $schema: "http://json-schema.org/schema#",
+    definitions: {
+      "debt-types": {
```

Comment:
We are defining this but not sure if we are using it.

Reply:
Actually, we don't